tensorflow/core/ops/dataset_ops.cc

```diff
@@ -464,6 +464,18 @@ REGISTER_OP("TFRecordDataset")
       return shape_inference::ScalarShape(c);
     });
 
+REGISTER_OP("SamplingDataset")
```

Comment:
move this definition to `experimental_dataset_ops.cc`

Reply:
fixed in https://github.com/tensorflow/tensorflow/pull/25384/commits/fa11069c3e85ffbcae19994bdba09c30eee0a6de#diff-1e0c16b2f31425b70be6193fc64b7949R504